backend/modules/prices/services.py

```
@@ -55,10 +60,126 @@ def get_instrument_price_history(
             data=data, min_price=min_price, max_price=max_price
         )
 
-    def _parse_time_interval(self, value: str) -> YFinanceTimeInterval:
-        try:
-            return YFinanceTimeInterval(value)
-        except ValueError as e:
-            raise InvalidTimeIntervalException(
-                f"Invalid time interval, valid intervals are: {", ".join([ti.value for ti in YFinanceTimeInterval])}"
-            ) from e
+
+type PriceUpdateHandler = Callable[[dict[str, float]], None]
```

Pylint (3.3.7)

[error] Parsing failed: 'invalid syntax (modules.prices.services, line 64)'

(E0001)